app/javascript/components/shared/helpers.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export const preserveLines = string => `${string}`.split("\n").map((line, idx) => <p key={idx}>{line}</p>);
```

Comment:
strange interpolation and maybe jsx fragment move into
`map((line, idx) => (
  <p> key={idx}>{line}</p>
))`